pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/housekeeping/schedule/DataSyncJob.java

```diff
@@ -92,7 +92,8 @@ public void execute(JobExecutionContext ctx) throws JobExecutionException {
             TransactionWrapper.useTxn(TransactionWrapper.DB.APIS, handle -> run(handle, exporter));
             log.info("Completed executing DataSyncJob");
         } catch (Exception e) {
-            throw new JobExecutionException(e, false);
+            log.error("Exception executing DataSyncJob", e);
```

Comment:
This will generate a alert to pepper-prod-alerts . Since we are trying to fix this issue seen only in Dev , may be a log.warn ?

Reply:
I think this should be an alert on all environments. If this fails it may likely mean that ES/DSM is out of sync with DSS (and in some enrollment cases, it is an error that must be fixed by hand). Since we have no other way of being aware of housekeeping problems, it is a bug to *not* alert.

At least some other job executors also log at error level. (Did not check them all.)

Reply:
Job runs every 10seconds. Hopefully next run will sync it. unless data issue or something and error keeps recurring.
Any way lets go with error and see if we get any error alerts in prod.